common/utils/src/main/java/com/splunk/sdk/utils/ApplicationInfoUtils.kt

```diff
@@ -56,11 +56,24 @@ class ApplicationInfoUtils {
                 return null
             }
 
-            return applicationInfo.metaData?.getString(SPLUNK_UUID_MANIFEST_KEY)?.takeIf {
-                it.isNotEmpty()
-            } ?: run {
-                Logger.e(TAG, "Application MetaData bundle is null or does not contain the UUID")
-                null
+            val metaData = applicationInfo.metaData
```

Comment:
Let’s choose a consistent naming convention for metadata. There’s still a mix of `metadata` and `metaData` being used.